src-backend/api/tests/utils/helpers.py

```diff
@@ -0,0 +1,2 @@
+def add_token_to_header(self, token):
```

Comment:
may as well make this more helpful by making it take in a request object and user